pkg/collector/corechecks/cluster/orchestrator/orchestrator_processor.go

```diff
@@ -40,6 +40,7 @@ func processCronJobList(cronJobList []*batchv1beta1.CronJob, groupID int32, cfg
 		if orchestrator.SkipKubernetesResource(cronJob.UID, cronJob.ResourceVersion, orchestrator.K8sCronJob) {
 			continue
 		}
+		redact.RemoveLastAppliedConfigurationAnnotation(cronJob.Annotations)
```

Comment:
If we move it early from the annotations map
e.g. `Annotations["kubectl.kubernetes.io/last-applied-configuration"] = "*********"`

we will also extract it the same way in later calls e.g. `extractMetadata` which converts above map to a slice